fastly/objectstorage/accesskeys/api_response.go

```diff
@@ -0,0 +1,27 @@
+package accesskeys
+
+// AccessKey is the API response structure for the create and describe operations.
+type AccessKey struct {
+	// AccessKey is an AccessKey identifier.
+	AccessKeyID string `json:"access_key"`
+	// SecretKey is the secret for the access key
+	SecretKey string `json:"secret_key"`
+	// Description is human readable description for the access key.
+	Description string `json:"description"`
+	// Permission is the permissions the key has.
+	Permission string `json:"permission"`
+	// Buckets is the list of buckets associated with the access key.
+	Buckets []string `json:"buckets"`
+	// CreatedAt is the timestamp associated with the creation of the access key.
+	CreatedAt string `json:"created_at"`
```

Comment:
Please update the type to `time.Time`. We use that type for all of the timestamp fields.